docs/how_to_guides.md

```diff
@@ -79,12 +79,12 @@ Python interpreter and Tavily Internet search are provided in the toolkit by def
 
 Example API call:
 ```bash
-curl --location 'http://localhost:8000/langchain-chat' \
+curl --location 'http://localhost:8000/v1/langchain-chat' \
 --header 'User-Id: me' \
 --header 'Content-Type: application/json' \
 --data '{
     "message": "Tell me about the aya model",
-    "tools": [{"name": "Python_Interpreter"},{"name": "Internet Search"},]
+    "tools": [{"name": "Python_Interpreter"},{"name": "Internet_Search"},]
```

Comment:
there's an additional comma here - this curl won't work

Reply:
ty